packages/scandipwa/src/component/ProductActions/ProductActions.style.scss

```diff
@@ -8,6 +8,7 @@
  * @package scandipwa/base-theme
  * @link https://github.com/scandipwa/base-theme
  */
+@import "../ProductInformation/ProductInformation.style.scss";
```

Comment:
Sorry but no, that won;t do

Mostly what you do, you import mixin, so set this mixin somewhere in app/style/base directory, and it will be accessable in this file